src/browser/BrowserCodeReader.ts

```diff
@@ -264,6 +264,10 @@ export default class BrowserCodeReader {
         }
     }
 
+    protected drawImageOnCanvas() {
```

Comment:
Parametize this method, so you don't have to worry about the protected accessor, then when you extend this class and overwrite this method, the base class will be passing the parameters you need lika injection. 🙂